src/annotator/integrations/vitalsource.js

```diff
@@ -292,22 +293,31 @@ export class VitalSourceContentIntegration {
       const bookContainer = /** @type {HTMLElement} */ (
         bookImage.parentElement
       );
+      const textLayer = this._textLayer;
 
       // Update the PDF image size and alignment to fit alongside the sidebar.
       // `ImageTextLayer` will handle adjusting the text layer to match.
       const newWidth = window.innerWidth - layout.width;
       const minWidth = 250;
 
-      if (layout.expanded && newWidth > minWidth) {
-        // The VS book viewer sets `text-align: center` on the <body> element
-        // by default, which centers the book image in the page. When the sidebar
-        // is open we need the image to be left-aligned.
-        bookContainer.style.textAlign = 'left';
-        bookImage.style.width = `${newWidth}px`;
-      } else {
-        bookContainer.style.textAlign = '';
-        bookImage.style.width = '';
-      }
+      preserveScrollPosition(() => {
```

Comment:
I think this method (`fitSideBySide`) could use a little helpful commenting. I spent a few minutes twisting up my brow before I realized:

* If this VitalSource document is a PDF, the logic for fitting side-by-side is defined here
* If this is an HTML document, delegate to the HTML integration's implementation of this

Now that I write it down, that seems dead simple. But I didn't spot it for a few minutes (maybe because the paydirt line is at the end of the method in the `else` branch) and as such, perhaps a comment would be helpful to speed up future devs.

I also find myself looking for a sort of "entry point"  for VitalSource document support that gives a brief summary of the big picture. All of the parts make sense but I'm not sure where a future dev would find the "linchpin". The doc comments for the `VitalSourceContentIntegration` class seem like a plausible starting point. Could be as simple as expanding that a touch (e.g. to make mention of PDFs)?

Reply:
I added some additional comments to clarify the behavior switching depending on the document type (PDF / EPUB), as well as some notes about PDF documents in the `VitalSourceContentIntegration` class docs.